blivet/udev.py

```diff
@@ -101,7 +113,7 @@ def resolve_devspec(devspec, sysname=False):
             if device_get_uuid(dev) == devspec[5:]:
                 ret = dev
                 break
-        elif device_get_name(dev) == devname or dev.sys_name == devname:
+        elif device_get_name(dev) == devname or dev["SYS_NAME"] == devname:
```

Comment:
I wonder if a new function called `device_get_sys_name` would be an improvement.